linux/Platform.c

```diff
@@ -230,8 +230,8 @@ void Platform_setZfsCompressedArcValues(Meter* this) {
    ZfsCompressedArcMeter_readStats(this, &(lpl->zfs));
 }
 char* Platform_getProcessEnv(pid_t pid) {
-   char procname[32+1];
-   xSnprintf(procname, 32, "/proc/%d/environ", pid);
+   char procname[128+1];
+   xSnprintf(procname, 128, PROCDIR "/%d/environ", pid);
```

Comment:
It's somehow bad style to repeat the buffer size, instead of using `sizeof(buffer)-1` as this creates multiple places that need updating if a buffer size is changed. Using the `sizeof` version avoids this pitfall.

Reply:
True, I just wanted to keep the patch small

Reply:
When both lines are touched there's not much diff here ;-)